src/bun.js/javascript.zig

```diff
@@ -1670,7 +1674,10 @@ pub const VirtualMachine = struct {
             ret.result = null;
             ret.path = result.path;
             return;
-        } else if (jsc_vm.module_loader.eval_script != null and strings.endsWithComptime(specifier, bun.pathLiteral("/[eval]"))) {
+        } else if (jsc_vm.module_loader.eval_script != null and
+            (strings.endsWithComptime(specifier, bun.pathLiteral("/[eval]")) or
+            strings.endsWithComptime(specifier, bun.pathLiteral("/[stdin]"))))
```

Comment:
I dig the comptime `bun.pathLiteral` helper.
###### _(less pretty alt: `std.fs.path.sep_str ++ "[eval]"`)_


👀 There's an unrelated line just above this that doesn't normalize the `/` like for platform like `bun.pathLiteral`. Maybe a future Win bug?
```
} else if (strings.hasPrefixComptime(specifier, "/bun-vfs/node_modules/")) {
```